### ISSUE TITLE
This does not parse html/dom 2 hyperscript, but to vtree instead

### ISSUE THREAD

**Issue by serapath** [LookLikeAPro/virtual-hyperscript-parser#1]
I was hoping to parse `"<div></div>"' to `"h('div')"`, but got the executed `h('div')` instead...
So maybe the module name is misleading?

**Comment by LookLikeAPro** [comment]
Good point. I may rename this at some point.
The functionality you are looking for is this: https://github.com/unframework/html2hyperscript

**Comment by serapath** [comment]
yes found it. found an even more interesting one with a similar bad name.
https://www.npmjs.com/package/vdom-raw

**Comment by jarmo** [comment]
I see it parsing into VirtualNode instead of a VTree. Which is it then? I hoped to use it for creating VTree-s out of DOM nodes for diffing, but to no avail. Unless I can somehow convert VirtualNode to VTree...
